assets/js/components/Work/ListItem.jsx

```diff
@@ -1,16 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
+import { buildImageURL } from "../../services/helpers";
 
 const WorkListItem = ({ work }) => {
   const fileSetsToDisplay = 5;
+  const imageIdToDisplay =
+    work.fileSets && work.fileSets[0] ? work.fileSets[0].id : "";
```

Comment:
Here's a little trick.  Usually when validating `array` type values, you'll check for length.  

`work.fileSets.length > 0 ? work.fileSets[0].id : ""`

Reply:
If fileset didn't exist, this would fail like "Cannot read property length of undef". I thought about the length and remembered that error. Let me know if you still want me to change this.

Reply:
``` 
var work = {};
undefined
work.filesets.length
VM245:1 Uncaught TypeError: Cannot read property 'length' of undefined
    at <anonymous>:1:15
```
Something like this.

Reply:
Ah, good point.  Make sure you're testing `fileSets` and not `filesets`.   If that's fine, then yea, include the check for `work.fileSets` but I still like the length check better than checking for a `[0]` element.